src/include/wiredtiger.in

```diff
@@ -3240,6 +3240,20 @@ int wiredtiger_unpack_uint(WT_PACK_STREAM *ps, uint64_t *up)
  * @{
  */
 
+/*! Permitted values of the \c type field. */
+typedef enum {
```

Comment:
Since this enum is removed from its context, I'd change the comment: `Permitted values of the \c type field in WT_CONFIG_ITEM` or something like that.  Since this is a documentation comment, it might be good to review how or if the documentation changes and whether it looks sensible.